### ISSUE TITLE
[Screen Reader-Social Embedded-Report Post]: TalkBack announces the state as "selected" for all radio buttons (selected/non-selected) while navigating through the list items via keyboard.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#19]
Priority: High

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Navigate to ellipses next to the post (topic) and press ENTER.
8. Press ENTER key on ""Report Post"" link and press ENTER to open ""Report an Issue"" page.
9. Using keyboard, navigate through the different issues in the list.
10. Verify the speech of talkback for all issues.

Expected
Talkback should announce the state as ""selected"" only for the selected radio buttons.So that screen reader users will not get confused to identify the selected radio buttons.

Actual
TalkBack announces the state as ""selected"" for all radio buttons (selected/non-selected) while navigating through the list items via keyboard.

MAS Rule: MAS4.2.2

Impact
If talkback announces the ""selected"" state for all radio buttons (selected/non-selected), then screen reader users will get confused to identify which radio buttons are selected and which are not.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
